testing/content/files/en-us/web/split_section_by_heading/index.html

```diff
@@ -0,0 +1,14 @@
+---
+title: Split section by heading
+slug: Web/Split_section_by_heading
+---
+
+<p>This page contains a few h2, h3 and h4 tags at root-level that will be direct linkified.</p>
+
+
+<h2>Some heading</h2>
+<h4>Foo buzz</h4>
+<h4 id="foo">Foo bar</h4>
+<h3>Another heading</h3>
+<div><p>stuff</p></div>
+<h4>Final heading</h4>
```

Comment:
Might be interesting to add a duplicate h4, i.e. a heading that also exists in another section.

Reply:
Agreed, it would be a good scenario to test